src/collective/ifttt/actions/easyform.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from collective.easyform.actions import Action
+from collective.easyform.actions import ActionFactory
+from collective.ifttt import _
+from interfaces.ifttt_easyform_adapter import IIFTTT
+from plone.supermodel.exportimport import BaseHandler
+from zope.interface import implementer
+
+
+@implementer(IIFTTT)
+class IFTTTTrigger(Action):
+    """Action Executor for easyform IFTTT adapter"""
+
+    def __init__(self, **kw):
+        for i, f in IIFTTT.namesAndDescriptions():
+            setattr(self, i, kw.pop(i, f.default))
+        # TODO solve this error
+        'ValueError: Field type collective.ifttt.actions.easyform.IFTTTTrigger ' \
+        'specified for field insdfd_sdf is not supported'
```

Comment:
@Shriyanshagro Would you be able to provide full stacktrace. It could be that fixing `IFTTTHandler = BaseHandler(IFTTTAction)` into `IFTTTHandler = BaseHandler(IFTTTTrigger)` would fix this, but I'm not completely sure.

Reply:
@datakurre no change after that little change. Below is the full Traceback
```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module plone.z3cform.layout, line 63, in __call__
  Module plone.z3cform.layout, line 47, in update
  Module plone.z3cform.fieldsets.extensible, line 65, in update
  Module plone.z3cform.patch, line 30, in GroupForm_update
  Module z3c.form.group, line 145, in update
  Module plone.app.z3cform.csrf, line 22, in execute
  Module z3c.form.action, line 98, in execute
  Module z3c.form.button, line 315, in __call__
  Module z3c.form.button, line 170, in __call__
  Module z3c.form.form, line 257, in handleAdd
  Module z3c.form.form, line 265, in createAndAdd
  Module plone.schemaeditor.browser.schema.add_field, line 88, in add
  Module zope.event, line 31, in notify
  Module zope.component.event, line 27, in dispatch
  Module zope.component._api, line 154, in subscribers
  Module zope.interface.registry, line 442, in subscribers
  Module zope.interface.adapter, line 607, in subscribers
  Module zope.component.event, line 36, in objectEventNotify
  Module zope.component._api, line 154, in subscribers
  Module zope.interface.registry, line 442, in subscribers
  Module zope.interface.adapter, line 607, in subscribers
  Module collective.easyform.subscribers, line 11, in updateActions
  Module collective.easyform.api, line 168, in set_actions
  Module plone.supermodel, line 37, in serializeSchema
  Module plone.supermodel, line 41, in serializeModel
  Module plone.supermodel.serializer, line 103, in serialize
  Module plone.supermodel.serializer, line 69, in writeField
ValueError: Field type collective.ifttt.actions.easyform.IFTTTTrigger specified for field ksdjf is not supported
```